app/assets/stylesheets/custom.css.scss

```diff
@@ -268,6 +275,14 @@ table td {
   font-color: white;
 }
 
+.twitter-login {
+  align: center;
+  margin-left: 25%;
+  margin-right: 25%;
+  text-align: center;
```

Comment:
Properties should be ordered align, margin-left, margin-right, padding, text-align